src/t11.c

```diff
@@ -0,0 +1,213 @@
+#include<stdio.h>
+#include<string.h>
+#include<stdlib.h>
+#include<math.h>
+#include<stdbool.h>
+typedef struct queue_num{
+    int n;
+    char c;
+    bool flag;
+    struct queue_num *left, *right;
+} queue_num;
+typedef struct stack{
+    char c;
+    struct stack *next;
+} stack;
+typedef struct stack_num{
+    int c;
+    struct stack_num *next;
+} stack_num;
+bool prior(char c){
+    if (c == '+' || c == '-')
+        return 0;
+    return 1;
+}
+void stack_push_num(stack_num *St, int c, int *len){
+    stack_num *new_elem = malloc(sizeof(stack_num));
+    new_elem->next = St->next;
+    St->next = new_elem;
+    new_elem->c = c;
+    (*len)++;
+}
+int stack_pop_num(stack_num *St, int *len){
+    if (len == 0) 
+        return 0;
+    else{
+        stack_num *del_elem = St->next;
+        int x = del_elem->c;
+        St->next = del_elem->next;
+        free(del_elem);
+        (*len)--;
+        return x;
+    }
+}
+void stack_push(stack *St, char c, int *len){
+    stack *new_elem = malloc(sizeof(stack));
+    new_elem->next = St->next;
+    St->next = new_elem;
+    new_elem->c = c;
+    (*len)++;
+}
+char stack_operation_get(stack *St){
+    return St->next->c;
+}
+void stack_pop(stack *St, int *len){
+    if (len == 0) 
+        return ;
+    else{
+        stack *del_elem = St->next;
+        St->next = del_elem->next;
+        free(del_elem);
+        (*len)--;
+    }
+}
+int func(stack *st, int *len){
+    int n = 0, c = 1;
+    while (*len){
+        n += ((st->next->c - '0') * c);
+        stack_pop(st, len);
+        c *= 10;
+    }
+    return n;
+}
+void add(queue_num *Tail, int num, char c, bool flag, int *len){
+    queue_num* elem = malloc(sizeof(queue_num));
+    if (flag)
+        elem->n = num;
+    else
+        elem->c = c;
+    elem->flag = flag;
+    queue_num *p = Tail->right;
+    Tail->right = elem;
+    p->left =  elem;
+    elem->left = Tail;
+    elem->right = p;
+    (*len)++;
+}
+void del(queue_num* head, int* len){
+    if (len == 0)
+	return;
+    else{
+	queue_num *p = head->left;
+	queue_num *p1 = p->left;
+	p1->right = head;
+	head->left = p1;
+	free(p);
+	(*len)--;
+    }
+}
+int main(){
+    char mas_c[100];
+    int len_queue = 0, len_op = 0, len_conversion_num = 0, len_num = 0;
+    bool flag = 0;
```

Comment:
Если используете `bool`, то нужно идти до конца:
```suggestion
    bool flag = false;
```